proto/neutron/transfer/v1/query.proto

```diff
@@ -9,14 +9,14 @@ option go_package = "github.com/neutron-org/neutron/v7/x/transfer/types";
 
 // Query provides defines the gRPC querier service.
 service Query {
-  // DenomTrace queries a denomination trace information.
-  rpc DenomTrace(ibc.applications.transfer.v1.QueryDenomTraceRequest) returns (ibc.applications.transfer.v1.QueryDenomTraceResponse) {
-    option (google.api.http).get = "/ibc/apps/transfer/v1/denom_traces/{hash}";
+  // Denoms queries all denominations
```

Comment:
are we gonna do proxy for deleted DenomTrace and DenomTraces calls?

Reply:
How hard is to do a proxy call to keep backwards compatibility?